react-app-rewired/headers/csps/defi/osmosis.ts

```diff
@@ -4,5 +4,8 @@ export const csp: Csp = {
   'connect-src': [
    process.env.REACT_APP_OSMOSIS_LCD_BASE_URL!,
     process.env.REACT_APP_OSMOSIS_IMPERATOR_BASE_URL!,
+    process.env.REACT_APP_OSMOSIS_ICON_BASE_URL!,
```

Comment:
these are missing from base config - this will work on your machine

Reply:
It looks like this actually isn't used anywhere - removing.

Reply:
It looks like this actually isn't used anywhere - removing.